packages/web3-atl-aspect/types/index.d.ts

```diff
@@ -86,7 +87,11 @@ export interface Options extends AspectOptions {
 export interface UpgradeOptions {
     data: string;
     properties?: KVPair[];
+    joinPoints?: JoinPoint[];
 }
+export type JoinPoint ="VerifyTx"|"PreTxExecute"|"PreContractCall"|"PostContractCall"|"PostTxExecute"|"PostTxCommit"|"Operation";
```

Comment:
I don't think `Operation` registration is necessary here, this should be enabled by default.